Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -0,0 +1,511 @@
+        QFrame* firstScreenFrame = new QFrame();
+
+        QWidget* firstScreenScrollWidget = new QWidget();
+        firstScreenScrollArea->setWidget(firstScreenScrollWidget);
+
+        QVBoxLayout* firstScreen = new QVBoxLayout(this);
+        firstScreen->setAlignment(Qt::AlignTop);
+        firstScreen->addWidget(firstScreenFrame);
+        firstScreenScrollWidget->setLayout(firstScreen);
+        QLabel* rightPaneHeader = new QLabel();
+        rightPaneHeader->setObjectName("rightPaneHeader");
+        rightPaneHeader->setText(tr("Please Choose a Gem Template"));
+        QLabel* rightPaneSubheader = new QLabel();
+        rightPaneSubheader->setObjectName("rightPaneSubheader");
+        rightPaneSubheader->setText(tr("Gems can contain assets such as scripts, animations, meshes, textures, and more."));
+        firstScreen->addWidget(rightPaneHeader);
+        firstScreen->addWidget(rightPaneSubheader);
+
+        LoadButtonsFromGemTemplatePaths(firstScreen);
+        m_formFolderRadioButton = new QRadioButton("Choose existing template");
+        m_formFolderRadioButton->setObjectName("createAGem");
+        m_radioButtonGroup->addButton(m_formFolderRadioButton);
+
+        m_gemTemplateLocation = new FormFolderBrowseEditWidget(tr("Gem Template Location*"), "");
+
+        firstScreen->addWidget(m_formFolderRadioButton);
+        firstScreen->addWidget(m_gemTemplateLocation);
+
+        return firstScreenScrollArea;
+    }
+
+    QScrollArea* CreateAGemScreen::CreateSecondScreen()
+    {
+        QScrollArea* secondScreenScrollArea = new QScrollArea(this);
+        secondScreenScrollArea->setWidgetResizable(true);
+        secondScreenScrollArea->setMinimumWidth(660);
+        secondScreenScrollArea->setObjectName("createAGemRightPane");
+        QFrame* secondScreenFrame = new QFrame();
+
+        QWidget* secondScreenScrollWidget = new QWidget();
+        secondScreenScrollArea->setWidget(secondScreenScrollWidget);
+
+        QVBoxLayout* secondScreen = new QVBoxLayout(this);
+        secondScreen->setAlignment(Qt::AlignTop);
+        secondScreen->addWidget(secondScreenFrame);
+        secondScreenScrollWidget->setLayout(secondScreen);
+
+        QLabel* secondRightPaneHeader = new QLabel();
+        secondRightPaneHeader->setObjectName("rightPaneHeader");
+        secondRightPaneHeader->setText(tr("Enter Gem Details"));
+        secondScreen->addWidget(secondRightPaneHeader);
+
+        m_gemDisplayName = new FormLineEditWidget(tr("Gem Display name*"), "");
+        QLineEdit* gemDisplayLineEdit = m_gemDisplayName->lineEdit();
+        gemDisplayLineEdit->setPlaceholderText(tr("The name displayed on the Gem Catalog"));
+        secondScreen->addWidget(m_gemDisplayName);
+
+        m_gemSystemName = new FormLineEditWidget(tr("Gem System name*"), "");
+        QLineEdit* gemSystemLineEdit = m_gemSystemName->lineEdit();
+        gemSystemLineEdit->setPlaceholderText(tr("The name used in the O3DE Gem System"));
```

Comment:
"system name"?  I don't think we use that language anywhere else and should just call this "Gem name" - also we should swap the order so this one comes first.
```suggestion
        m_gemName = new FormLineEditWidget(tr("Gem name*"), "");
        m_gemName->lineEdit()->setPlaceholderText(tr("The unique name for your gem that must a single word consisting of alphanumeric characters, '-' and '_'."));
       connect(m_gemName->lineEdit(), &QLineEdit::textChanged, [this]{ ValidateGemName(); });
```